packages/splitter/src/SplitterContainer.tsx

```diff
@@ -29,20 +30,27 @@ export const SplitterContainer: React.FC<ISplitterContainerProps> = props => {
 };
 
 SplitterContainer.propTypes = {
+  idPrefix: PropTypes.string,
   ariaLabel: PropTypes.string,
-  primaryPaneId: PropTypes.string.isRequired,
   children: PropTypes.func,
   render: PropTypes.func,
-  environment: PropTypes.oneOfType([
-    PropTypes.instanceOf(Document),
+  windowObject: PropTypes.oneOfType([
     PropTypes.shape({
-      addEventListener: PropTypes.func.isRequired,
-      removeEventListener: PropTypes.func.isRequired,
-      body: PropTypes.shape({
-        clientWidth: PropTypes.number.isRequired,
-        addEventListener: PropTypes.func.isRequired,
-        removeEventListener: PropTypes.func.isRequired
-      }).isRequired
+      scrollX: PropTypes.number.isRequired,
+      scrollY: PropTypes.number.isRequired,
+      document: PropTypes.oneOfType([
+        PropTypes.instanceOf(Document),
```

Comment:
I need to remove this as well for the same reasons why `window` failed in the SSR test.